website/docs/amiga_quick_start/amiga-quick-start.md

```diff
@@ -0,0 +1,58 @@
+---
+id: amiga-quick-start
+title: Quick Start Guide
+---
+
+### Welcome to the Amiga Quick Start Guide!
+
+Here will be the guide to everything you will need to know to start using your Amiga as soon as possible!
+
+:::info
+Things that will be included with Amiga
+- Two batteries
+- Battery charging kit
+- Debug cable
+:::
+
+## Batteries
+
+Be sure to insert both battery packs in their respective locations on the Amiga.
+
+![battery](https://user-images.githubusercontent.com/64480560/206290736-59ef9164-9ade-401c-80dc-569e77f088e5.gif)
+
+
+:::caution
+While operating the Amiga, be sure not to stand in front of the Amiga. Also it is important that it is on a flat surface!
```

Comment:
Maybe make this a bullet point list of key warnings and put it before any sub sections (i.e. before Batteries)